pyfr/integrators/controllers.py

```diff
@@ -49,6 +51,13 @@ def __init__(self, *args, **kwargs):
         # Delete the memory-intensive elements map from the system
         del self.system.ele_map
 
+    def ensure_tstep(self, tstep):
```

Comment:
I understand tstep to be the separation between invocations of the plugin, and ensure_tstep to construct a list of the times that the solver should hit, so the plugin can be invoked at the appropriate times. Therefore, could tstep be renamed dtinvoke and could ensure_tstep be renamed hit_dtinvokes?